golden_response/staging/get_pop_obs/moscow.json

```diff
@@ -242,82 +242,6 @@
         }
       ]
     },
-    "dc/p/119gy3zecnm61": {
```

Comment:
This population appears to have become the population "dc/p/89rd4chpd74r5" below. None of the PVs have changed though.

<img width="655" alt="Screen Shot 2020-08-20 at 5 22 22 PM" src="https://user-images.githubusercontent.com/18621425/90838537-f3a35580-e309-11ea-9b42-5b89e2290b41.png">

<img width="1540" alt="Screen Shot 2020-08-20 at 5 23 54 PM" src="https://user-images.githubusercontent.com/18621425/90838602-20f00380-e30a-11ea-9464-077c2a662f21.png">

Reply:
Figured out this is because of [Years 0 14] becoming [Years - 14]. So this and the next diff are expected.